iothub/device/src/Transport/AmqpIoT/AmqpIoTOutcome.cs

```diff
@@ -1,50 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Weird diff for this file and next. They appear below, but git didn't pick up it was just a rename.